courses/admin.py

```diff
@@ -48,6 +48,10 @@ class ProgramEnrollmentAdmin(admin.ModelAdmin):
     """Admin for ProgramEnrollment"""
 
     model = ProgramEnrollment
+    search_fields = ["user", "company", "Program"]
+    list_filter = ["active", "change_status"]
+    list_fields = ["user.email", "change_status"]
```

Comment:
Don't understand the intent, couldn't find `list_fields` documented in core Django, or am I missing something. Also, `user__email` would be the correct way to reference the user's email attribute.